feature-libs/order/components/order-history/order-history.component.spec.ts

```diff
@@ -223,6 +260,31 @@ describe('OrderHistoryComponent', () => {
     expect(elements.length).toEqual(2);
     expect(component.sortType).toEqual('byDate');
   });
+
+  it('should display PO Number & Cost Center', () => {
```

Comment:
We should also test that columns don't appear when there is no `PONumber`. These expects could be added to an existing test.

Reply:
line 265 ~ 271 is checking for it without PONumber and
line 273 ~ 285 is fo with PONumber
Thanks~